dolphinscheduler-dao/src/main/resources/org/apache/dolphinscheduler/dao/mapper/ProcessDefinitionMapper.xml

```diff
@@ -180,4 +180,16 @@
         where id = #{processDefinitionId}
         and version = #{version} limit 1
     </select>
+
+    <select id="existDefByProjectCodeAndDefineName" resultType="java.lang.Boolean">
+        select 1
+        from t_ds_process_definition pd
+        JOIN t_ds_user u ON pd.user_id = u.id
```

Comment:
It seems that we just need to join the `t_ds_project` table, am I right?
```sql
select 1 from t_ds_process_definition pd
JOIN t_ds_project p ON pd.project_code = p.code
WHERE p.code = #{projectCode}
and pd.name = #{processDefinitionName}
```

Reply:
fixed.